Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -350,6 +350,42 @@ void AODProducerWorkflowDPL::fillTrackTablesPerCollision(int collisionID,
   }
 }
 
+void AODProducerWorkflowDPL::fillIndexTablesPerCollision(const o2::dataformats::VtxTrackRef& trackRef, const gsl::span<const GIndex>& GIndices)
+{
+  //  for (int src = GIndex::NSources; src--;) {
+  for (int src : {GIndex::Source::MFTMCH, GIndex::Source::MCH, GIndex::Source::MFT}) {
+    int start = trackRef.getFirstEntryOfSource(src);
+    int end = start + trackRef.getEntriesOfSource(src);
+    for (int ti = start; ti < end; ti++) {
+      auto& trackIndex = GIndices[ti];
+      if (GIndex::includesSource(src, mInputSources)) {
+        if (src == GIndex::Source::MFT) {
+          if (trackIndex.isAmbiguous() && mGIDToTableMFTID.find(trackIndex) != mGIDToTableMFTID.end()) {
+            continue;
+          }
+
+          mGIDToTableMFTID.emplace(trackIndex, mIndexMFTID);
+          //          mIndexTableMFT.emplace(trackIndex.getIndex(), mIndexMFTID);
+          mIndexTableMFT[trackIndex.getIndex()] = mIndexMFTID;
```

Comment:
don't you need to fill mGIDToTableMFTID and mIndexTableMFT also for the MFT contributors of global forward tracks? See comment on L 538.

Reply:
I am not sure I understand the comment. Here we fill the maps for all MFT tracks, including contributors to global forward tracks